audio/phonon-backend-vlc/Portfile

```diff
@@ -18,20 +32,43 @@ homepage            http://projects.kde.org/projects/kdesupport/phonon/phonon-vl
 distname            phonon-backend-vlc-git
 
 if {${os.platform} eq "darwin"} {
-    depends_lib-append  path:lib/libvlc.dylib:libVLC
+    if {${os.major} <= 15} {
+        variant vlc2 description {build against libVLC2} {}
+    }
+}
+if {[variant_exists vlc2] && [variant_isset vlc2]} {
+    depends_lib-append \
+                    path:libexec/vlc2/lib/pkgconfig/libvlc.pc:libVLC2
+    configure.pkg_config_path-append "${prefix}/libexec/vlc2/lib/pkgconfig"
+} else {
+    depends_lib-append \
+                    path:libexec/vlc3/lib/pkgconfig/libvlc.pc:libVLC
+    configure.pkg_config_path-append "${prefix}/libexec/vlc3/lib/pkgconfig"
+}
+pre-configure {
+    configure.env-append PKG_CONFIG_PATH=[join ${configure.pkg_config_path} ":"]
+    configure.pkg_config_path
```

Comment:
Why is this happening? Doesn't MacPorts base already set `PKG_CONFIG_PATH` for you based on `configure.pkg_config_path`?

Reply:
I haven't tried to understand in detail what's going on here; my guess would be that "base" doesn't set PKG_CONFIG_PATH appropriately when multiple configure.pkg_config_path is a list with more than 1 element.

I searched for examples of configure.pkg_config_path use in released portfiles and found this in port:avahi:

```
# see https://trac.macports.org/ticket/55054
pre-configure {
    configure.env-append PKG_CONFIG_PATH=[join ${configure.pkg_config_path} ":"]
    configure.pkg_config_path
}
```

Adapted it this accepted solution, my problem went away and I moved on to more interesting things...

Reply:
MacPorts base was fixed to support multi-value configure.pkg_config_path values in 2.5.0. That's what https://trac.macports.org/ticket/55054 was about. I've removed the workaround from avahi in 0e2f3c814bd966e6bb7f21beb439001f173d7867 and will wee what I can do to find any other ports still using that workaround.

So this should not be necessary, but you said your problem went away. What problem were you seeing without using this workaround?

Reply:
I've removed the workaround too now. Sorry, I cannot remember exactly what problem I was seeing. In fact, I see no (more) evidence of there being multiple paths in the list. So maybe there was no problem at all and I just copied the entire snippet as "this is how you have to use `configure.pkg_config_path`.